README.md

```diff
@@ -3,8 +3,6 @@
 
 This library is a Rust implementation of the [Prefixed API Key](https://github.com/seamapi/prefixed-api-key) typescript library. Though its interface differs slightly from the typescript version, this library provides the same set of features and functionality as the typescript version.
 
-⚠️ This library is still a work in progress.
-
```

Comment:
The project has settled enough that it's safe to remove the wip warning I think 😁